extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1451,6 +1451,36 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'piStats': {
+      'requests': {
+          'host': 'https://events.pi-stats.com',
+          'basePrefix': '${host}/eventsamp/?' +
+          'e=PageLoad&' +
+          'pid=${property}&' +
+          'url=${ampdocUrl}&' +
+          'cnt=${cntId}&' +
+          'lang=${language}&' +
+          'ref=${documentReferrer}&' +
+          'id=${clientId(piStatsDEVICEID)}&'+
+          'ua=${userAgent}&'+
+          'ctype=web&'+
+          'blang=${browserLanguage}&'+
+          'v=2.0&'+
+          'dist=Javascript'
+      },
+      'triggers': {
+          'defaultPageview': {
+              'on': 'visible',
+              'request': 'pageview',
```

Comment:
there is no pageview request defined.

Reply:
could you please elaborate?

Reply:
On the visible event, AMP will try to find the request named `pageview`.  However in your `requests` config, you only specify `host` and `basePrefex` but not `pageview` this will lead to an error.

Please refer to the [request doc](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/amp-analytics.md#requests) for more details.

Reply:
Okk made the required changes

Reply:
Could you please validate?

Reply:
Yes, this looks good.